k8s/elasticsearch/chart/elasticsearch/templates/manifests.yaml.template

```diff
@@ -76,11 +76,40 @@ spec:
           value: "{{ .Release.Name }}-elasticsearch-svc"
         - name: BACKUP_REPO_PATH
           value: ""
+        - name: NGINX_PROMETHEUS_ENABLED
+          value: "true"
         ports:
+        - name: prometheus
+          containerPort: 9108
+          protocol: TCP
         - name: http
           containerPort: 9200
         - name: tcp-transport
           containerPort: 9300
+        {{ if .Values.metrics.enabled }}
```

Comment:
This part should have two less spaces to be treated as container, not as a port.

Reply:
Also, it should be placed after first container description.